lib/Provider/UnsplashAPI.php

```diff
@@ -22,19 +22,21 @@
 
```

Comment:
This file (and a lot of other files) are full of alignment/linting issues, please run something of the lines of `php-cs-fixer`, at least for the new files. Maybe there needs to be a cleanup follow up
This makes reviewing even harder